config/locales/notices/en.yml

```diff
@@ -48,11 +48,13 @@ en:
         continue: Continue sign in
         message_html: For your security, in %{time_left_in_session} we will cancel your
           sign in.
+        sr_message_html: You will be signed out in %{time_left_in_session}. Select "keep me signed in" to stay logged in.
```

Comment:
looks like the `partially_signed_in` has different text than `signed_in`, should the screen reader text match?